src/libraries/System.Drawing.Common/tests/Imaging/PropertyItemTests.cs

```diff
@@ -30,7 +8,80 @@ namespace System.Drawing.Imaging.Tests
     {
-        private const string PngFile = "16x16_nonindexed_24bit.png";
+        private const int PropertyTagLuminanceTable = 0x5090;
+
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/22221", TestPlatforms.AnyUnix)]
+        [ConditionalTheory(Helpers.IsDrawingSupported)]
+        [InlineData(1)]
+        [InlineData(0)]
+        [InlineData(-1)]
+        public void Id_Set_GetReturnsExpected(int value)
+        {
+            using var bitmap = new Bitmap(Helpers.GetTestBitmapPath("nature24bits.jpg"));
+            PropertyItem item = bitmap.GetPropertyItem(PropertyTagLuminanceTable);
+            item.Id = value;
+            Assert.Equal(value, item.Id);
+
+            // Set same.
+            item.Id = value;
+            Assert.Equal(value, item.Id);
+        }
+
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/22221", TestPlatforms.AnyUnix)]
+        [ConditionalTheory(Helpers.IsDrawingSupported)]
+        [InlineData(1)]
+        [InlineData(0)]
+        [InlineData(-1)]
+        public void Len_Set_GetReturnsExpected(int value)
+        {
+            using var bitmap = new Bitmap(Helpers.GetTestBitmapPath("nature24bits.jpg"));
+            PropertyItem item = bitmap.GetPropertyItem(PropertyTagLuminanceTable);
+            item.Len = value;
+            Assert.Equal(value, item.Len);
+
+            // Set same.
+            item.Len = value;
+            Assert.Equal(value, item.Len);
+        }
+
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/22221", TestPlatforms.AnyUnix)]
+        [ConditionalTheory(Helpers.IsDrawingSupported)]
+        [InlineData(1)]
+        [InlineData(0)]
+        [InlineData(-1)]
+        public void Type_Set_GetReturnsExpected(short value)
+        {
+            using var bitmap = new Bitmap(Helpers.GetTestBitmapPath("nature24bits.jpg"));
+            PropertyItem item = bitmap.GetPropertyItem(PropertyTagLuminanceTable);
+            item.Type = value;
+            Assert.Equal(value, item.Type);
+
+            // Set same.
+            item.Type = value;
+            Assert.Equal(value, item.Type);
+        }
+
+        public static IEnumerable<object[]> Value_Set_TestData()
+        {
+            yield return new object[] { null };
+            yield return new object[] { Array.Empty<byte>() };
+            yield return new object[] { new byte[] { 1, 2, 3 } };
+        }
+
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/22221", TestPlatforms.AnyUnix)]
+        [ConditionalTheory(Helpers.IsDrawingSupported)]
+        [MemberData(nameof(Value_Set_TestData))]
+        public void Value_Set_GetReturnsExpected(byte[] value)
+        {
+            using var bitmap = new Bitmap(Helpers.GetTestBitmapPath("nature24bits.jpg"));
+            PropertyItem item = bitmap.GetPropertyItem(PropertyTagLuminanceTable);
+            item.Value = value;
+            Assert.Same(value, item.Value);
+
+            // Set same.
```

Comment:
What is this exercising? I mean double setting to the same value?

Reply:
Its something we do in winforms where we test setting a property to the same value to make sure things are idempotent.

Removed